src/SnapKeyring.test.ts

```diff
@@ -643,6 +694,38 @@ describe('SnapKeyring', () => {
       );
       spy.mockRestore();
     });
+    const isNotAllowedOrigin = async (
```

Comment:
Any reason why this one has been moved here now?

Reply:
Got messed up in the merged! Fixed here [2373c78](https://github.com/MetaMask/eth-snap-keyring/pull/300/commits/2373c783c0184c8ac5d40a9ecf63645cea63af99)